tests/test_templates.py

```diff
@@ -181,7 +196,7 @@ def test_yes_no_template_process_output(self):
             yes_answer: {label_field: ["news", "sports"], class_field: ["news"]},
         }
         for expected_processed_output, outputs in processed_output_to_outputs.items():
-            processed = template.process_outputs(outputs)
+            processed, _ = template.outputs_to_target_and_references(outputs)
```

Comment:
Should add checks for references.

Reply:
I would do it in a separate PR as this one does not aim to add things just to reorganize code